crates/experimentation_platform/src/db/models.rs

```diff
@@ -1,12 +1,21 @@
 use crate::db::schema::*;
 use chrono::{DateTime, NaiveDateTime, Utc};
 
-use diesel::{query_builder::QueryId, Insertable, Queryable, QueryableByName, Selectable};
+use diesel::{
+    query_builder::QueryId, Insertable, Queryable, QueryableByName, Selectable,
+};
 use serde::{Deserialize, Serialize};
 use serde_json::Value;
 
 #[derive(
-    Debug, Clone, Copy, PartialEq, Deserialize, Serialize, diesel_derive_enum::DbEnum, QueryId,
+    Debug,
+    Clone,
+    Copy,
+    PartialEq,
+    Deserialize,
+    Serialize,
+    diesel_derive_enum::DbEnum,
+    QueryId,
```

Comment:
@mahatoankitkumar you can get rid of QueryId here, I was trying to solve something and added it by mistake.